src/components/WinesList/WineList.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { DataGrid, GridColDef } from '@mui/x-data-grid'
+import { Wine } from '../../interfaces/wine_interface'
+
+export const WineList: React.FC<{
+    winesList: Wine[]
+}> = ({ winesList }) => {
+
+    const columns: GridColDef[] = [
+        {
+            field: "id",
+            headerName: "ID",
+            headerAlign: "center",
+            align: "center"
+        },
+        {
+            field: "name",
+            headerName: "Name",
+            headerAlign: "center",
+            align: "center",
+            width: 300
+            
+        },
+        {
+            field: "description",
+            headerName: "Description",
+            headerAlign: "center",
+            align: "center",
+            width: 450
+        },
+        {
+            field: "rating",
+            headerName: "Rating",
+            headerAlign: "center",
+            align: "center"
+        },
+        {
+            field: "quantity",
+            headerName: "Quantity",
+            headerAlign: "center",
+            align: "center"
+        }
+    ]
+
+    const rows = winesList
+
+    return (
+        <>  
```

Comment:
???

Reply:
Is this like a new way to write <Fragment> ?

Reply:
Not a new way to write Fragment, but ultimately performs the same function as Fragment. If you're trying to render multiple elements, you can group them together using `<>, </>`. The result on the browser when you inspect are `<div>`

Reply:
nice try 😝 : "<> is the shorthand tag for React.Fragment which allows us to group a list of elements without wrapping them in a new node."

Reply:
I was close, but good to know they are `<>, </>` are `Fragment`